Dockerfile

```diff
@@ -1,38 +1,64 @@
 FROM ubuntu:18.04
 
-RUN mkdir /data
+RUN mkdir /data /package
 
-ENV BBLFSH_HOSTNAME=tmexp_bblfshd \
-  BBLFSH_PORT=9432 \
-  GITBASE_HOSTNAME=tmexp_gitbase \
-  GITBASE_PORT=3306 \
-  GITBASE_USERNAME=root \
-  GITBASE_PASSWORD=
+ARG DEBIAN_FRONTEND=noninteractive 
+
+ENV BBLFSH_HOSTNAME tmexp_bblfshd
+ENV BBLFSH_PORT 9432
+ENV GITBASE_HOSTNAME tmexp_gitbase
+ENV GITBASE_PORT 3306
+ENV GITBASE_USERNAME root
+ENV GITBASE_PASSWORD ""
+ENV ARTM_SHARED_LIBRARY /usr/local/lib/libartm.so
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 RUN apt-get update \
-  && apt-get install -y dialog apt-utils curl \
+  && apt-get install -y --no-install-recommends \
+  apt-utils \
+  ca-certificates \
+  curl \
+  locales \
+  && echo "en_US.UTF-8 UTF-8" > /etc/locale.gen \
+  && locale-gen \
   && apt-get clean \
   && rm -rf /var/lib/apt/lists/*
 
-RUN curl -sSL https://get.docker.com/ | sh 
-
-RUN apt-get update \
-  && apt-get install -y python3-pip python3-dev \
-  && cd /usr/local/bin \
-  && ln -s /usr/bin/python3 python \
-  && apt-get remove -y .*-doc .*-man >/dev/null \
-  && apt-get autoremove --purge -y \ 
+RUN curl -sSL https://get.docker.com/ | sh \
+  && apt-get update \
+  && apt-get install -y --no-install-recommends \
+  build-essential \
+  cmake \
+  git \
+  libboost-all-dev \
+  make \
+  python3-dev \
+  python3-pip \
+  && ln -s /usr/bin/python3 /usr/local/bin/python \
+  && pip3 install numpy scipy pandas protobuf tqdm wheel \
+  && git clone --branch stable \
+  https://github.com/bigartm/bigartm.git /opt/bigartm \
+  && cd /opt/bigartm \
+  && git reset --hard 3c0cbc4ac08cae37f00754815c105fde930ff379 \
```

Comment:
Did you check this is needed ? From [here](https://github.com/bigartm/bigartm/commits/stable) it seems it changes nothing, as the last commit reverted the one before, so the module should be in the same state at the HEAD and at commit `3c0cbc4ac08cae37f00754815c105fde930ff379`

Reply:
ah okay. actually, I just realized that this commit hash is actually for the release v0.10.0,
so you can just do `git clone  --branch v0.10.0 https://github.com/bigartm/bigartm.git /opt/bigartm`

Reply:
Good point!